### ISSUE TITLE
[Web] Page up/down for scrolling

### ISSUE THREAD

**Issue by odiferousmint** [wireapp/wire-webapp#7021]
You should be able to scroll up and down via the page up and down keys.

Edit: I know that this is not a discussion board, and the reason I submitted my feature request is that it is strictly limited to the desktop version of Wire.

**Comment by bennycode** [comment]
Scrolling up and down using the keyboard keys is now possible when opening a conversation. 🥳
